README.md

```diff
@@ -342,15 +317,13 @@ An entity is a representation of anything that needs to be monitored. It can be
 #### Properties
 * `subscriptions` An array of subscriptions. If no subscriptions are provided,
 it defaults to an entity-specific subscription list: `[entity:{ID}]`.
-* `class_` **required** the entity type, must be either `agent` or `proxy`.
-* `organization` the name of the organization the entity belongs to, defaults to `default`
-* `environment` the name of the environment the entity belongs to, defaults to `default`
+* `entity_class` **required** the entity type, must be either `agent` or `proxy`.
```

Comment:
if its required then maybe defaulting to `agent` makes sense? @portertech @grepory whats the rationale for not having a sane default?

Reply:
entity_class is sort of weird and annoying, but removing it from the API would have been too costly for GA. It *is* configurable by the user (if you could file a docs ticket please), but the class names have special meaning.

“agent” is reserved for sensu agents.

Setting the value of entity_class to proxy will create a proxy entity.

Otherwise, the field isn’t required or the user can use it to indicate a type of entity. E.g. “lambda” “switch” etc.

Reply:
Thanks for the clarification @grepory that helps. A docs PR is open https://github.com/sensu/sensu-docs/pull/929

Reply:
since behavior is unchanged from before, this is just a doc change I'm thinking we can handle the discussion of behavior adjustments for the resource as a smaller item on separate issue or PR?